### PR TITLE
tools/cli: fixes the event-id

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

**Comment by NoahSaso** [comment]
assuming it's ok for `lavs-apis` to depend on `layer-climb`, this works now! much cleaner using the task queue event enum, nice

**Review by dakom — APPROVED** [review]

File: tools/cli/src/commands/task_queue.rs [hunk 1]
```diff
@@ -89,9 +89,10 @@ impl TaskQueue {
             )
             .await?;
 
-        let event: TaskCreatedEvent = CosmosTxEvents::from(&tx_resp)
+        let event: cosmwasm_std::Event = CosmosTxEvents::from(&tx_resp)
             .event_first_by_type(TaskCreatedEvent::NAME)?
-            .try_into()?;
+            .into();
+        let event: TaskCreatedEvent = event.try_into()?;
```
Comment (ethanfrey) [inline tools/cli/src/commands/task_queue.rs:95]: This looks better. The cli can obviously care about Climb and all. And this translation makes sense

**Review by ethanfrey — COMMENTED** [review]
Better this way

File: packages/apis/src/events/task_queue_events.rs [hunk 2]
```diff
@@ -66,5 +67,12 @@ macro_rules! define_task_queue_event {
                 event
             }
```
[the remark this answers is not in the document]
Comment (gangov) [reply packages/apis/src/events/task_queue_events.rs]: I've created #107 for this

**Pull request merged**